website/contributing/release-candidate-patch.md

```diff
@@ -29,13 +29,19 @@ git checkout -b <release-branch>
 git cherry-pick <commit-hash>
 ```
 
-### 2. Test the current changes
+### 2. Bump monorepo packages
+
+- Update packages in the monorepo that were modified by the cherry picks by running `yarn bump-all-updated-packages`. All the package bumps should be on the patch level. Read more about the script and how they work [here](./release-updating-packages).
+
+- CI will take care of releasing the new versions on npm once you push the changes to the `0.XX-stable` branch.
```

Comment:
Following the first point, I'll be more prescriptive here:

```suggestion
Update all packages in the monorepo that were modified by the cherry picks. You can do it by running:
    ```
    yarn bump-all-updated-packages # All the package bumps should be on the patch level
    git push origin 0.XX-stable
    ```
After pushing, the CI will take care to publish the new packages automatically.

**For 0.71: **
// Add instructions here

** For 0.70: **
For version 0.70, packages must be released manually by a Meta engineer. 
```